core/lib/refinery/plugin.rb

```diff
@@ -1,5 +1,16 @@
 module Refinery
   class Plugin
+    META = {
```

Comment:
I've been lurking on this pull request with interest and I think putting the icon as a config line in the engine makes a lot of sense and is better than the META constant. If there is no icon specified in config, use the default.